src/functions/installRepoPackage.ts

```diff
@@ -19,3 +19,5 @@ export const installDependencies = async () => {
   });
   console.log("Dependencies installed successfully.");
 };
+
+export const installRepoPackage = installDependencies("npm", ["install"]);
```

Comment:
I abstracted the commands passed in, but didn't use it in the end. Maybe I should undo this work, idk.